edx_toggles/toggles/internal/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+This module includes all code related to feature toggles. Remember to import publicly available classes and functions
+in toggles/__init__.py.
```

Comment:
Could you move this to a README.rst instead?

Reply:
Are you sure? This comment will only be read by developers, so it would make sense to keep it as near as possible to the code (IMHO).

Reply:
**tl;dr:** You can leave this as-is.

**Longer:**
I'm not sure. :) I used to hate `__init__.py` files because you never knew if they were empty or not, so you never knew whether to look in them.  Now that we are on a newer version of Python, can we remove them if they are empty?  Assuming so, I have less of an issue with docs being in this file. :)

Reply:
I am marking this as resolved so I don't have to keep seeing it.